examples/private_connection/README.md

```diff
@@ -0,0 +1,71 @@
+
+1. Forseti infrastructure
+   * CloudSQL Database
+   * Forseti Server GCS Bucket
+   * Forseti Client GCS Bucket
+   * Forseti Client VM
+   * Forseti Server IAM Service Account
+   * Forseti Client IAM Service Account
+2. DNS and NAT configurations for private communication between forseti server and GCP services
+3. Firewall rules to control ingress and egress connectivity
+
+## Requirements
+
+Before this example can be used, you have to ensure that the following pre-requisites are fulfilled:
+
+* Terraform is [installed](#software-dependencies) on the machine where the script is executed.
+* GCP Credentials should be [setup](#setup-credentials)
+* (OPTIONAL) Service APIs should be [active](#enable-apis) on the project that you plan to deploy the example
+
+The [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory) can be used to provision projects with the correct APIs active.
+
+### Software Dependencies
+
+#### Terraform and Plugins
+
+* [Terraform](https://www.terraform.io/downloads.html) 0.12+
+* [Terraform Provider for GCP](https://www.terraform.io/docs/providers/google/index.html) v2.9+
+
+### Setup Credentials
+
+When [configuring Terraform credentials for GCP](https://www.terraform.io/docs/providers/google/guides/getting_started.html#adding-credentials) you should take care that the identity has proper roles on the example's project. In addition to the [roles](https://github.com/forseti-security/terraform-google-forseti#iam-roles) required for the core module function, the identity must have these roles for this example.
+
+* roles/iam.serviceAccountAdmin
+* roles/iam.serviceAccountKeyAdmin
+* roles/compute.networkAdmin
+* roles/resourcemanager.projectIamAdmin (only required if `service_account` is set to `create`)
+
+### Enable APIs
+
+In order to operate the following APIs should be active on the example's project:
+
+* Compute Engine API - compute.googleapis.com
+* Cloud Resource Manager API - cloudresourcemanager.googleapis.com
+* Service Usage API - serviceusage.googleapis.com
+* Service Networking API - servicenetworking.googleapis.com
+* Cloud DNS API - dns.googleapis.com
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Inputs
+
+You can setup the inputs by renaming and editing the `terraform.tfvars2` file to `terraform.tfvars` or use alternative methods.
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| project\_id | The ID of an existing Google project where Forseti will be installed | string | n/a | yes | org\_id | GCP Organization ID that Forseti will have purview over | string | n/a | yes |
+| domain | Domain that is associated with the GCP organisation | string |  | yes |
+| region | Region where forseti subnetwork will be deployed | string | `"us-central1"` | no |
+| block_egress | Controls whether the forseti infrastructure components can communicate to internet | string | `true` | no |
```

Comment:
I would default it to `false`. People will start with set it to `false` to install Forseti.

Reply:
And the type is `string`? Is it auto-generated by the `make` command?

Reply:
it is copy/paste error.